CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.3.0
+
+## 04/29/2020
+
+- Migrated from using CARTO tables for VIIRS data to the Elasticsearch datasets generated by geotrellis.
+
```

Comment:
Did the response pattern change? I remember, you mentioned that not all download URLs will be available any more. Please document changes here.

Reply:
Ok, added a note in the change log, and the swagger already included this information